LINK/etc/default/evm.properties

```diff
@@ -0,0 +1,47 @@
+RAILS_ENV=production
+APPLIANCE=true
+# Force ExecJS to use node
+EXECJS_RUNTIME='Node'
+# workaround for virtual memory spike observed with RHEL6
+MALLOC_ARENA_MAX=1
+# Location of certificates and provider keys (miq-password and appliance console)
+KEY_ROOT=/var/www/miq/vmdb/certs
+
+# Force locale
+LANGUAGE=en_US.UTF-8
+LANG=en_US.UTF-8
+LC_CTYPE=en_US.UTF-8
+
+# Postgres (appliance console, PgAdmin)
+APPLIANCE_PG_CTL=/usr/bin/pg_ctl
+APPLIANCE_PG_MOUNT_POINT=/var/lib/pgsql
+APPLIANCE_PG_DATA=/var/lib/pgsql/data
+APPLIANCE_PG_SERVICE=postgresql
+APPLIANCE_PG_PACKAGE_NAME=postgresql-server
+
+APPLIANCE_SOURCE_DIRECTORY=/opt/manageiq/mangeiq-appliance
+APPLIANCE_TEMPLATE_DIRECTORY=/opt/manageiq/mangeiq-appliance/TEMPLATE
+BUNDLE_GEMFILE=/var/www/miq/vmdb/Gemfile
+GEM_HOME=/opt/manageiq/manageiq-gemset
+GEM_PATH=/opt/manageiq/manageiq-gemset:/usr/share/gems:/usr/local/share/gems
```

Comment:
The /opt/manageiq is productizable, so it can't be hardcoded

Reply:
We might be able to make the rpm emit this file.  Or perhaps we keep it as a template in the template dir and generate from these based on actual system values?

cc @bdunne

Reply:
Cool. I'll put this section back into the rpm.
Will need to make sure this changes in all appliance rpms going forward.